subschemas/substrate-chain/src/resolvers/Query/parachains.ts

```diff
@@ -60,30 +68,52 @@ export function parachain(_: Record<string, never>, params: {id: string}, {api}:
 
 const extractParachainData = async (
   api: Context['api'],
+  accountsService: AccountsService,
```

Comment:
The `AccountsService` is not used in the function body.
Are you keeping this for later work as mentioned in the PR description?

Reply:
Ah, I was thinking about using it to fetch the accounts but I'll remove it from the PR, as it's not needed here.